src/Lorentz/Contracts/BaseDAO.hs

```diff
@@ -33,7 +33,11 @@ baseDaoContract config@Config{..} = defaultContract $ contractName cDaoName $ do
     , #cTransfer_ownership /-> transferOwnership
     , #cAccept_ownership /-> acceptOwnership
     , #cPropose /-> propose config
-    -- , #cVote /-> vote --TODO
+    , #cProposalMetadata /-> proposalMetadata
+    , #cVote /-> vote
+    , #cSetVotingPeriod /-> setVotingPeriod
```

Comment:
AFAIU this entrypoint's name will be `setVotingPeriod`, however, our specification says it should be `set_voting_period`. Same applies to some other entrypoints.

Reply:
Got this, Ivan. I've fixed this accordingly.